clouddq/lib.py

```diff
@@ -124,18 +131,22 @@ def create_rule_binding_view_model(
         high_watermark_filter_exists=high_watermark_filter_exists,
         bigquery_client=bigquery_client,
     )
+
     sql_string = template.render(configs)
+    failed_records_sql_string = failed_records_template.render(configs)
     if debug:
         configs.update({"generated_sql_string": sql_string})
+        configs.update({"failed_records_sql_string": failed_records_sql_string})
```

Comment:
move this out of debug if statement. If debug: logger.info(pformat(configs)) only. configs should always be created